src/audio/crossover/crossover.c

```diff
@@ -11,8 +11,8 @@
 #include <sof/audio/format.h>
 #include <sof/audio/pipeline.h>
 #include <sof/audio/ipc-config.h>
+#include <sof/math/crossover_common.h>
```

Comment:
I'm not a fan of moving crossover_common.h under math. See my rationale at https://github.com/thesofproject/sof/issues/8630#issuecomment-1864940768 

Not blocking on this though if other reviewers are ok.

Reply:
this is a intermediate solution, Seppo and I will continue try to figure out better refactor solution for this, @singalsu @kv2019i

Reply:
move it to include/module.